configs/qvhighlight/clip_slowfast_moment_detr_qvhighlight_pretrain.yml

```diff
@@ -7,7 +7,7 @@ train_path: data/qvhighlight/subs_train.jsonl
 eval_path: data/qvhighlight/highlight_val_release.jsonl
 eval_split_name: val
 v_feat_dirs: ['features/QVHighlight/clip', 'features/QVHighlight/slowfast']
-t_feat_dir: features/QVHighlight/clip_text_subs_train/moment_detr
+t_feat_dir: features/QVHighlight/clip_text_subs_train
 t_feat_dir_eval: features/QVHighlight/clip_text
```

Comment:
I guess that currently, t_feat_dir_eval is not used in the code. Is it used?

Reply:
I apologize that i did not reflect the codes which is used t_feat_dir_eval. I will add that codes.